Formula/kfctl.rb

```diff
@@ -0,0 +1,20 @@
+class Kfctl < Formula
+  desc "Kubeflow CLI tool"
+  homepage "https://www.kubeflow.org"
+  url "https://github.com/kubeflow/kubeflow/releases/download/v0.5.1/kfctl_v0.5.1_darwin.tar.gz"
+  version "0.5.1"
```

Comment:
This is in the URL already

Reply:
You mean the URL? I think Homebrew didn't like the “v” in front of the actual version number.

Reply:
I've not seen it go wrong in such cases but sure.